src/emulator/modules/SceCtrl/SceCtrl.cpp

```diff
@@ -27,6 +27,8 @@
 #include <algorithm>
 #include <array>
 
+#include <util/log.h>
```

Comment:
Please preserve include type order, this is a Vita3K file so it should go above `psp2/*` and below `SceCtrl.h` (with newlines above and below).

Our codebase follows this, but I don't think it's included in the Style Guide in the wiki. Probably should be.